fuzzers/035-iob-ilogic/generate.py

```diff
@@ -19,13 +19,43 @@ def handle_data_width(segmk, d):
     if 'DATA_WIDTH' not in d:
         return
 
-    if d['DATA_RATE'] == 'DDR':
-        return
-
-    for opt in [2, 3, 4, 5, 6, 7, 8, 10, 14]:
+    # It appears several widths end up with the same bitstream pattern.
+    # This groups those widths together for documentation.
```

Comment:
this looks like some kind of optimization in bits usage. Since 5 and 7 widths cannot be used in DDR mode, the SDR bit is most probably used to set the width configuration in the device

Reply:
I like that theory.  I originally documented this feature as "W_ODD", which would totally 100% correlate with SDR modes.